tests/Doctrine/Tests/Mocks/ConnectionMock.php

```diff
@@ -4,9 +4,19 @@
 
 class ConnectionMock extends \Doctrine\DBAL\Connection
 {
-    private $_fetchOneResult;
+    /**
+     * @var DatabasePlatformMock
```

Comment:
Unlike the previous platform mock, this type cannot be replaced with the abstract class (no action required).